modules/processor/src/main/java/uk/gov/hmcts/reform/migration/CaseMigrationProcessor.java

```diff
@@ -36,30 +48,53 @@ public void processSingleCase(String userToken, String caseId) {
             log.error("Case {} not found due to: {}", caseId, ex.getMessage());
             return;
         }
-        if (dataMigrationService.accepts().test(caseDetails)) {
-            updateCase(userToken, caseDetails.getId(), caseDetails.getData());
-        } else {
-            log.info("Case {} already migrated", caseDetails.getId());
-        }
+        updateCase(userToken, caseDetails.getCaseTypeId(), caseDetails);
     }
 
-    public void processAllCases(String userToken, String userId) {
-        coreCaseDataService.fetchAll(userToken, userId).stream()
-            .filter(dataMigrationService.accepts())
-            .forEach(caseDetails -> updateCase(userToken, caseDetails.getId(), caseDetails.getData()));
+    public void processAllCases(String userToken, String userId, String caseType) {
+        coreCaseDataService.fetchAll(userToken, userId, caseType).stream()
+            .forEach(caseDetails -> updateCase(userToken, caseType, caseDetails));
```

Comment:
This is using the old SQL approach, but I don't think is being used by anything now anyway (as it shouldn't be, since the new `migrateCases()` method below is the one which is invoked by the Runner and uses ES)? So we should be able to delete this.

Reply:
removed this